src/components/UI/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,20 @@
+'use client'
+import { PropsCheckbox } from '@/types/Checkbox'
+
+export default function Checkbox({ label, isChecked, onCheck }: PropsCheckbox) {
+
+  return (
+    <>
+      <label className="inline-flex items-center">
```

Comment:
It's better when `label` has `htmlFor` attribute with `id` of input,  as this further improves accessibility.

P.S. Consider adding aria-checked to communicate the state of the checkbox to screen readers explicitly.
(`aria-checked={isChecked}` for your `input`)

Reply:
fixed